src/commands/cmd_json.cc

```diff
@@ -204,11 +205,59 @@ class CommandJsonArrLen : public Commander {
   }
 };
 
+class CommanderJsonArrIndex : public Commander {
+ public:
+  Status Execute(Server *svr, Connection *conn, std::string *output) override {
+    redis::Json json(svr->storage, conn->GetNamespace());
+
+    std::vector<ssize_t> result;
+    int start = -1;
```

Comment:
Why here it is `-1` instead of `0`?

Reply:
negative start index also need to be normalized, so default value should be 0